webrtc.nimble

```diff
@@ -33,5 +33,5 @@ proc runTest(filename: string) =
   exec excstr & " -r " & " tests/" & filename
```

Comment:
Why is it necessary? I don't recall seeing it before.

Reply:
What do you mean? It's necessary to run the tests.

Reply:
Didn't notice it before in libp2p